From roessler@does-not-exist.org Wed Nov  7 11:49:56 2001
Date: Wed, 7 Nov 2001 11:49:56 +0100
From: Thomas Roessler <roessler@does-not-exist.org>
To: mutt-dev@mutt.org
Subject: To those shipping patches (v2)
MIME-Version: 1.0
Content-Type: text/plain; format=flowed
Status: RO
Content-Length: 1273
Lines: 34

You folks have convinced me that the old patchlist was almost the 
right way to go for mutt identifying what patches have been applied.

Thus, I've implemented this scheme (and will commit it to the CVS 
in a moment): There's a new file called PATCHES in the source tree, 
which will be empty in the official distribution.

This file's format is one patch ID per line.  Patch IDs should be 
the same as the file names used for distributing patches.  The 
format for these file names should be this: 

     patch-<version>.<initials>.<patch-description>.<patchlevel>

That is, Vsevolod's NNTP patch for mutt-1.3.42 could be named like 
this:

     patch-1.3.42.vvv.nntp.1

 From PATCHES, patchlist.c will be automatically generated.  In 
order to properly construct PATCHES, please include the following 
chunk with your patch, replacing <your-id-here> by your patch ID.

------------------------------snip------------------------------
--- PATCHES~    Tue Nov  6 19:59:33 2001
+++ PATCHES     Tue Nov  6 19:59:42 2001
@@ -1,0 +1 @@
+<your-id-here>
------------------------------snip------------------------------

The patch IDs will be displayed when mutt is run with the 'v' 
command line switch.

-- 
Thomas Roessler                        http://log.does-not-exist.org/

